test/dlc_tests/container_tests/bin/testMXNetHVD

```diff
@@ -44,7 +49,6 @@ echo "Train a CNN model using Horovod with MXNet on MNIST data. You can follow p
 if [ ${RETURN_VAL} -eq 0 ]; then
   GPU_AMOUNT=`nvidia-smi -L | wc -l`
   if [ $GPU_AMOUNT -ge 8 ]; then GPU_AMOUNT=8; fi
```

Comment:
Can we confirm removing this does not hamper the test execution?

Reply:
you mean removing GPU_AMOUNT? why do remove this?